### PR TITLE
Feature/someip feeder

### PR DESCRIPTION
**Pull request by int0x27**
SOME/IP to KUKSA.VAL DataBroker feeder

- Uses vsomeip library to subscribe and receive an example "Wiper" Service events.
- Provides vsomeip example wiper service for local (on single host, unix sockets only) and remote (udp) testing.
- Provides helpers for KUKSA.VAL Databroker integration.
- Provides generic someip client for receiving SOME/IP Events.
- Example someip configurations and scripts for modifying those configurations.

**NOTE:** dockerisation of someip feeder is not ready, only generic someip examples are dockerised as POC

### CODE REVIEW

File: someip2val/docker/vsomeip/examples/notify-sample.cpp [hunk 1]
```diff
@@ -0,0 +1,305 @@
+// Copyright (C) 2014-2017 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
```
Comment (SebastianSchildt) [inline someip2val/docker/vsomeip/examples/notify-sample.cpp:1]: We need an Eclipse Apache header here if we argue the example is rather documentation or we need to do an CQ for MPL if we consider this a modified someip ocde

File: someip2val/docker/vsomeip/examples/subscribe-sample.cpp [hunk 2]
```diff
@@ -0,0 +1,207 @@
+// Copyright (C) 2014-2017 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
```
Comment (SebastianSchildt) [inline someip2val/docker/vsomeip/examples/subscribe-sample.cpp:1]: same as above.

Also in any case we need to add Eclipse KUKSA (c) above to not give the impression BMW is responsible for whatever bad things we might have done to their code

**Comment by lukasmittag** [comment]
Works for me!

File: someip2val/docker/vsomeip/examples/notify-sample.cpp (hunk 1 above)
Comment (int0x27) [reply someip2val/docker/vsomeip/examples/notify-sample.cpp]: I've removed modified examples, they had some dumps added, nothing important

**Review by SebastianSchildt — APPROVED** [review]
lgtm

**Pull request merged**